README.md

```diff
@@ -192,14 +192,12 @@ into one composite project. The easiest way is to use the [Base Starter Gradle](
 example project.
 
 1. Clone the Base Starter Gradle project and open it in Intellij
-2. Remove the entire `buildscript{}` block and the `apply plugin:"com.vaadin"` line and
-   uncomment the `id("com.vaadin")` line.
-3. Create a `settings.gradle` file containing the following line: `includeBuild("/home/mavi/work/vaadin/vaadin-gradle-plugin")`
+2. Create a `settings.gradle` file containing the following line: `includeBuild("/home/mavi/work/vaadin/vaadin-gradle-plugin")`
```

Comment:
```suggestion
2. Create a `settings.gradle` file containing the following line: `includeBuild("/path/to/your/plugin/project/vaadin-gradle-plugin")`
```

Just a random idea how to improve the docs here, not related to the patch itself.

Reply:
Thanks, done :+1: